src/vis/vis.js

```diff
@@ -744,6 +744,11 @@ var Vis = View.extend({
       }
     }
 
+    vizjson.overlays.push({
+      type: 'attribution',
+      order: 5
```

Comment:
This mechanism of adding overlays feels a bit hacky (I know other overlays are added this way). Why is the order 5?